.changeset/curly-steaks-mate.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/backend-app-api': minor
+'@backstage/backend-common': minor
+'@backstage/config-loader': minor
+---
+
+Allow an additionalConfig to be provided to loadBackendConfig that fetches config values during runtime.
```

Comment:
```suggestion
---
'@backstage/backend-app-api': minor
'@backstage/backend-common': minor
---

Allow additional configuration data to be provided to `loadBackendConfig` through the new `additionalConfigs` option.
```

`config-loader` is no longer involved, and figure we could message this a bit neater